addons/account_edi_ubl_cii/models/account_edi_common.py

```diff
@@ -664,15 +667,25 @@ def _import_fill_invoice_line_taxes(self, journal, tax_nodes, invoice_line_form,
                 ('type_tax_use', '=', journal.type),
                 ('amount', '=', amount),
             ]
-            tax_excl = self.env['account.tax'].search(domain + [('price_include', '=', False)], limit=1)
-            tax_incl = self.env['account.tax'].search(domain + [('price_include', '=', True)], limit=1)
-            if tax_excl:
-                inv_line_vals['taxes'].append(tax_excl.id)
-            elif tax_incl:
-                inv_line_vals['taxes'].append(tax_incl.id)
-                inv_line_vals['price_unit'] *= (1 + tax_incl.amount / 100)
-            else:
+
+            tax = False
+            if predictive_bill_module_installed:
```

Comment:
Easier to write/read, and never any additional query run (fine since it is tested)
```suggestion
            if hasattr(invoice_line_form, '_predict_specific_tax'):
```